unstructured/partition/common.py

```diff
@@ -183,6 +186,69 @@ def normalize_layout_element(
         )
 
 
+def set_hierarchy_by_indentation(  # Add exception handling for elements without coordinates
+    elements: List[Element],
+) -> List[Element]:
+    left = []
+    right = []
+    coordinates = []
+    for ele in elements:
+        coordinates.append(getattr(ele.metadata.coordinates, "points")[0])
+
+    coords_np = np.asarray(coordinates)
+    median = np.median(coords_np)
+
+    # Split text from multicolumn page in left and right lists
+    # Will not work for more than 2 columns
+    for ele in elements:
+        if ele.metadata.coordinates.points[0][0] <= median:
+            left.append((ele, "left"))
+        else:
+            right.append((ele, "right"))
+
+    # Assign parent id to indented text
+    prev_ele = []
+    i = 0
+    for ele, col in left:
+        if prev_ele:
+            if (
+                ele.metadata.coordinates.points[0][0] - prev_ele.metadata.coordinates.points[0][0]
+            ) > 0.79 and isinstance(ele, Title):
```

Comment:
What's the significance of 0.79 here? Is that like a minimum distance that if the left edges of two elements are more than that far apart we consider it "indented"?

Reply:
0.79 is the threshold for if something is on the left or right side of the page.